src/Ocelot/DownstreamUrlCreator/Middleware/DownstreamUrlCreatorMiddleware.cs

```diff
@@ -68,9 +75,9 @@ public async Task Invoke(HttpContext httpContext)
                 {
                     downstreamRequest.AbsolutePath = GetPath(dsPath);
                     var newQuery = GetQueryString(dsPath);
-                    downstreamRequest.Query = string.IsNullOrEmpty(downstreamRequest.Query)
-                        ? newQuery
-                        : MergeQueryStringsWithoutDuplicateValues(downstreamRequest.Query, newQuery, placeholders);
+                    downstreamRequest.Query = string.IsNullOrEmpty(downstreamRequest.Query)
+                        ? newQuery
+                        : MergeQueryStringsWithoutDuplicateValues(downstreamRequest.Query, newQuery, placeholders);

```

Comment:
Zero diff! No code changes. But GitHub highlighted!
Line endings?
**Please, use Visual Studio!** We have special editor settings for mixed line endings!
Please rebase onto develop, which should help to remove ghost changes.

Reply:
I am using Visual studio, and line-ending in all files are warned they are mixed, I believe we need to make a pass on line endings. but I will rebase to fix this

Reply:
Yes, rebase please! It should help to remove unclear changes.